modules/coo-troubleshooting-ui-plugin-using.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+
+// * observability/cluster_observability_operator/ui_plugins/troubleshooting-ui-plugin.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="coo-troubleshooting-ui-plugin-using_{context}"]
+= Using the {coo-full} troubleshooting UI plugin
+
+.Prerequisites
+* You have access to the {product-title} cluster as a user with the `cluster-admin` cluster role. If your cluster version is 4.17+, you can access the troubleshooting UI panel from the Application Launcher {launch}.
+* You have logged in to the {product-title} web console.
+* You have installed {product-title} Logging, if you want to visualize correlated logs.
+* You have installed {product-title} Network Observability, if you want to visualize correlated netflows.
+* You have installed the {coo-full}.
```

Comment:
Suppose should be You have installed the {coo-full} Troubleshooting UI plugin

Reply:
Done